doc/examples/anim09/anim09.sh

```diff
@@ -6,7 +6,7 @@
 # path to simulate a constant velocity at the given altitude, with synthetic
 # banking as we turn to follow the path.  We use the 30 arc second global relief
 # grid and overlay a few labels for named features.
-# Grid:   @earth_relief_30s.grd
```

Comment:
@PaulWessel I copied all the comments in the script to `doc/rst/source/animations/anim09.rst`. See https://docs.generic-mapping-tools.org/dev/animations/anim09.html for how it looks like. Perhaps we should remove the comments from the script and only maintain the one in the reST file.

Reply:
Yes I agree, and of course I just added a PR for 7 and 8 as well.